docs/Displaying a User Interface/Alerts/index.md

```diff
@@ -137,7 +137,7 @@ alert.duration = @(4000);
 ##### Swift
 ```swift
 // Duration timeout is in milliseconds
-alert.duration = 4000 as NSNumber
+alert.duration = NSNumber(4000)
```

Comment:
There are two other usages of `as NSNumber` in this guide. 

There are also three other usages of `as NSNumber` in the **Retrieving Vehicle Data** guide,

Reply:
Weird, they didn't show up originally. Fixed.